### ISSUE TITLE
Which pretrained model were used for Facenet512?

### ISSUE THREAD

**Issue by AnthraX1** [serengil/deepface_models#6]
Hi,

https://github.com/davidsandberg/facenet/blob/master/README.md listed two pretrained models:

## Pre-trained models
| Model name      | LFW accuracy | Training dataset | Architecture |
|-----------------|--------------|------------------|-------------|
| [20180408-102900](https://drive.google.com/open?id=1R77HmFADxe87GmoLwzfgMu_HY0IhcyBz) | 0.9905        | CASIA-WebFace    | [Inception ResNet v1](https://github.com/davidsandberg/facenet/blob/master/src/models/inception_resnet_v1.py) |
| [20180402-114759](https://drive.google.com/open?id=1EXPBSXwTaqrSC0OhUdXNmKSh9qJUQ55-) | 0.9965        | VGGFace2      | [Inception ResNet v1](https://github.com/davidsandberg/facenet/blob/master/src/models/inception_resnet_v1.py) |

Obviously the 20180402-114759 has a higher LFW accuracy. I would like to know which model was facenet512_weights.h5 converted from?

**Comment by serengil** [comment]
Facenet512 is the one with 99.6 score.
